### PR TITLE
DC-0 Combined maps

### PR DESCRIPTION
**Pull request by zonca**
- create solar dipole notebook
- combine maps script
- combine maps script

### CODE REVIEW

File: 202305_dc0/combine_maps.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        "dust_d10",
+        "synchrotron_s4",
+        "freefree_f1",
+        "ame_a1",
+        "co_co3",
+    ]
+    + extragalactic,
+    "combined_foregrounds_lowcomplexity": [
+        "dust_d9",
+        "synchrotron_s5",
```
Comment (keskitalo) [inline 202305_dc0/combine_maps.py:21]: Just double-checking: does `synchrotron_s4` have higher complexity than `synchrotron_s5`? For all other components the complexity seems to increase with the version number.

File: 202305_dc0/combine_maps.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
+import os
+import healpy as hp
+import numpy as np
+from astropy.table import QTable
+from pathlib import Path
+
+extragalactic = ["cib_cib1", "ksz_ksz1", "tsz_tsz1", "radio_rg1"]
```
Comment (keskitalo) [inline 202305_dc0/combine_maps.py:7]: There are no dusty galaxies?

**Review by keskitalo — APPROVED** [review]
I don't see any issue.  Please check the two comments I made.  The script generates the following sky maps:

1. unlensed CMB (including CMB dipole)
2. CMB lensing signal
3. low complexity diffuse foregrounds + extragalactic
4. medium complexity diffuse foregrounds + extragalactic
5. high complexity diffuse foregrounds + extragalactic

Can @jdborrill confirm we want to map each of the 5 maps separately?

File: 202305_dc0/combine_maps.py (hunk 2 above)
Comment (zonca) [reply 202305_dc0/combine_maps.py:7]: yes `rg1` are the radio galaxies

File: 202305_dc0/combine_maps.py (hunk 1 above)
Comment (zonca) [reply 202305_dc0/combine_maps.py:21]: thanks, this is a bug, I launched the script again, will let you know when this is transferred to nersc

**Comment by zonca** [comment]
@keskitalo I fixed the script, executed again and transferred the files to
NERSC.

Il lun 10 lug 2023, 06:33 Reijo Keskitalo ***@***.***> ha
scritto:

File: 202305_dc0/combine_maps.py (hunk 1 above)
Comment (zonca) [reply 202305_dc0/combine_maps.py]: Ok transferred

**Comment by jdborrill** [comment]
As I understand it, we generate all of the components separately and then
combine them into the 5 maps for delivery.

We will deliver the 5 maps to the community (through the data portal), but
only use 3 of them in DC0 (unlensed cmb + dipole, lensing perturbation,
medium complexity foregrounds)

Julian

**Pull request merged**